isis/tests/Fixtures.cpp

```diff
@@ -122,6 +122,7 @@ namespace Isis {
     TempTestingFiles::SetUp();
 
     std::ifstream isdFile("data/defaultImage/defaultCube.isd");
+    std::ifstream isdFile1("data/defaultImage/defaultCube.isd");
```

Comment:
I don't see an isdFile1 being used in the DefaultCube. Do we need this here?

Reply:
Good catch! This doesn't need to be in there. I'll update it.